docs/topics/mpp/mpp-connect-to-apis.md

```diff
@@ -72,7 +72,30 @@ internal actual fun writeLogMessage(message: String, logLevel: LogLevel) {
 The main rules regarding expected and actual declarations are:
 * An expected declaration is marked with the `expect` keyword; the actual declaration is marked with the `actual` keyword.
 * `expect` and `actual` declarations have the same name and are located in the same package (have the same fully qualified name).
-* `expect` declarations never contain any implementation code.
+* `expect` declarations never contain any implementation code and are abstract by default.
+* In interfaces, functions in `expect` declarations cannot have bodies, but their `actual` counterparts can be non-abstract
+and have a body, allowing the inheritors not to implement a particular function.
```

Comment:
I bit complicated.
Does it work this way: It allows the inheritors not to implement....  ?